packages/react/src/progress-indicator/ProgressIndicator.tsx

```diff
@@ -34,6 +34,19 @@ export const ProgressIndicator = ({
 				items.length
 		  } steps completed`;
 
+	const hasExplicitActiveDeclared =
+		items.filter(({ isActive }) => isActive).length > 0;
+
+	// If no explicit active items are declared, the 'doing' status item receives active state by default
+	const itemsWithDefaultActive: ProgressIndicatorItem[] =
+		hasExplicitActiveDeclared
+			? items
+			: items.map((item) => ({
+					isActive: item.status === 'doing',
+					// placing rest after isActive so people can overwrite the isActive value to false
```

Comment:
Not sure I understand this. If the progress indicator is displayed, that means a user should be on one of the pages in that progress list. Why would we allow consumers to set `isActive` to `false` (on a `doing` step) so nothing is selected?

Reply:
My aim was to lean on the side of being flexible rather restrictive. Maybe someone would want to display a preview of what the steps would be if the user chose a specific option.
Like "if you select this option, these are the steps you would need to complete". This would be a use case for displaying the steps component with no active state.

Reply:
The progress indicator only appears in the left side when you're on a page that is in a child flow of a task list step, so I can't see we'd ever need to not have an active state

Reply:
Actually I think you are right, in the use case I described above, you would use the `"todo"` status, not the `"doing"` status. And if you really want to not have an active state you would use the "started" status and not apply `isActive`.

Reply:
switched them around now so `isActive: item.status === 'doing'` overwrites `...item`